shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/config/impl/postgresql/PostgreSQLContainerConfigurationFactory.java

```diff
@@ -17,27 +17,37 @@
 
 package org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.impl.postgresql;
 
-import org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.StorageContainerConfiguration;
-
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
 import java.util.Collections;
 import java.util.Map;
 
-public class DefaultPostgreSQLContainerConfiguration implements StorageContainerConfiguration {
+/**
+ * PostgreSQL container configuration factory.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class PostgreSQLContainerConfigurationFactory {
+    
+    /**
+     * Create new instance of postgresql container configuration.
+     * 
+     * @return created instance
+     */
+    public static PostgreSQLContainerConfiguration newInstance() {
+        return new PostgreSQLContainerConfiguration(getCommands(), getContainerEnvs(), getMountedResources());
+    }
     
-    @Override
-    public String[] getCommands() {
+    private static String[] getCommands() {
         String[] commands = new String[1];
         commands[0] = "-c config_file=/etc/postgresql/postgresql.conf";
         return commands;
     }
     
-    @Override
-    public Map<String, String> getEnvs() {
+    private static Map<String, String> getContainerEnvs() {
```

Comment:
Please do not use abbreviation in method name

Reply:
done